lib/tasks/database_updates/create_cards_for_old_parameter_channels.rake

```diff
@@ -0,0 +1,19 @@
+namespace :database_updates do
+  task :create_cards_for_old_parameter_channels, [:start_id] => :environment do |t, args|
+    p args[:start_id]
+    completed_kyc = UpholdConnection.where(is_member: true).where("id > ?", args[:start_id]).order(id: :asc)
+
+    puts "Queueing up create cards for #{completed_kyc.size} users"
+    completed_kyc.each do |connection|
+      parameters = JSON.parse(connection.uphold_access_parameters)
+      next if parameters["scope"].include?("cards:write")
+      parameters["scope"].concat(" cards:write")
```

Comment:
Looks good, I forget if scope is a string or not; if `parameters["scope"]` is an array, then consider dropping the leading space ex. `parameters["scope"].concat("cards:write")`

if it is a string then everythings good 👍

Reply:
Looks like it's a string 😄 
```json
{"access_token":"","token_type":"bearer","scope":"cards:read user:read cards:write","server":"https://api.uphold.com"}
```